backend/src/Logging.hs

```diff
@@ -0,0 +1,46 @@
+module Logging where
+
+import Control.Monad.Logger (Loc, LogLevel (..), LogSource, LogStr, defaultLoc)
+import System.Log.FastLogger
+  ( FormattedTime,
+    LogType' (..),
+    ToLogStr (..),
+    defaultBufSize,
+    newTimeCache,
+    newTimedFastLogger,
+    simpleTimeFormat,
+  )
+
+type Logger = Loc -> LogSource -> LogLevel -> LogStr -> IO ()
+
+type LogLevelFilter = LogSource -> LogLevel -> Bool
```

Comment:
Another type [expected by the library](https://hackage.haskell.org/package/monad-logger-0.3.40/docs/Control-Monad-Logger.html#v:filterLogger). Nice helper function for log filtering.